File: mobile/app/src/main/java/com/auratherm/mobile/model/Zone.kt
package com.auratherm.mobile.model

import com.google.gson.annotations.SerializedName

data class Zone(
    val id: String,
    val name: String,

    // Залишаємо назви min_temp та max_temp, щоб вони збігалися з кодом Activity
    @SerializedName("min_temp")
    val min_temp: Double,

    @SerializedName("max_temp")
    val max_temp: Double,

    @SerializedName("store_id")
    val storeId: String
)

File: frontend/src/types/zone.ts
export interface SensorBasic {
  id: string;
  name: string;
  status: string;
}

export interface Zone {
  id: string;
  name: string;
  min_temp: number;
  max_temp: number;
  store_id: string;
  _count?: {
    sensors: number;
  };
  sensors?: SensorBasic[]; // Це поле приходить, коли викликається функція getById
}
